### PR TITLE
zeebe api tutorial

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Zeebe API docs tutorial -- relates to https://github.com/camunda/developer-experience/issues/142 and https://github.com/camunda/camunda-api-tutorials/pull/8.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:29]:
```suggestion
To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. You will need to add your `ZEEBE_CLIENT_ID`, `ZEEBE_CLIENT_SECRET`, `ZEEBE_BASE_URL`, and `ZEEBE_AUDIENCE`, which is `zeebe.camunda.io` in a Camunda 8 SaaS environment. For example, your audience may be defined as `ZEEBE_AUDIENCE=zeebe.camunda.io`.
```

We don't need this one anymore!

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:170]:
```suggestion
4. In your terminal, run `node cli.js zeebe unassign <task id>`, where `<task id>` is the Zeebe user task ID.
```

The email address isn't needed in the unassign call.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
A couple small details! Nice work.

**Comment by christinaausley** [comment]
This is ready for final review 🥳

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (akeller) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:41]: 💡 Since Zeebe user tasks are a bit of a new concept, it might be worth mentioning briefly in this tutorial that task assignment happened in Tasklist previously. We don't have to add this now, but it might be something to get broader feedback on.

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (akeller) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:93]: Add endpoint? Did you mean assign?

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (akeller) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:113]: 💡 You could consider having the user assign their own email address if they want to see it in the UIs.

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (akeller) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:119]: Non-blocking feedback: I was surprised that the `action` wasn't required in the API reference docs.

> A custom action value that will be accessible from user task events resulting from this endpoint invocation. If not provided, it will default to "assign".

I did see that this is consistent (if no `action` is provided, it will default) everywhere except when unassigning a user. I'll let you decide if you want to call this out and how you want to note it. Alternatively, you can wait for feedback on this.

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (pepopowitz) [reply docs/apis-tools/zeebe-api-rest/tutorial.md:41]: In addition, I think we should call out that they can capture a Zeebe user task ID, which they will use to assign/unassign in this API, from Tasklist....but it has to be a _Zeebe_ task, not a legacy task. 

We could link them to [this doc](https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks/), which describes the difference between zeebe user tasks & legacy user tasks. It's an important distinction that I personally overlooked, and I think it's likely that readers will do the same.

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/zeebe-api-rest/tutorial.md:58]:
```suggestion
3. Within the function, you must first generate an access token for this request, so your function should now look like the following:
```

non-blocking, this feels like a more accurate verb to what we're doing at this point in the tutorial.

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (pepopowitz) [reply docs/apis-tools/zeebe-api-rest/tutorial.md:113]: 🗒️ if you were wondering, like me, what would happen if one assigned a task via the API to a user not in their organization, the answer is:

It will work just fine.

Comment (pepopowitz) [reply docs/apis-tools/zeebe-api-rest/tutorial.md:113]: Also, I don't have a specific suggestion, but this might also be a good place to specify where they got a Zeebe user task ID from. ("The ID you've captured from Tasklist" or something like that)

**Review by pepopowitz — CHANGES_REQUESTED** [review]
I think we should do a little more to help users understand the distinction between Zeebe user tasks and legacy tasks. Nice work, almost there!

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (christinaausley) [reply docs/apis-tools/zeebe-api-rest/tutorial.md:93]: Adjusted in latest commit 👍

File: docs/apis-tools/zeebe-api-rest/tutorial.md
Comment (christinaausley) [reply docs/apis-tools/zeebe-api-rest/tutorial.md:113]: Adjusted in latest commit 👍

**Comment by christinaausley** [comment]
This is ready for final review 👍

**Review by akeller — APPROVED** [review]
I won't have time to walk through this tutorial myself, but I'm happy with how it's written! Thank you for incorporating my feedback! 🚀

**Review by pepopowitz — APPROVED** [review]

**Pull request merged**
